apps/conte/src/app/components/authentication/authentication.component.ts

```diff
@@ -75,53 +75,52 @@ export class AuthenticationComponent implements OnInit {
 
   login() {
     this.buttonState = 'loading';
-    const credentials : LoginCreds = {
+    const credentials: LoginCreds = {
       email: this.f.email.value,
       password: this.f.password.value,
     };
     this.authService
-    .userLogin(credentials)
-    .then((resp) => {
-      localStorage.setItem('token', resp.data.token);
-      localStorage.setItem('user_email', resp.data.email);
-      localStorage.setItem('is_verified', resp.data.is_verified);
-      localStorage.setItem('user_id', resp.data.user_id);
-      this.buttonState = 'static';
+      .userLogin(credentials)
+      .then((resp) => {
+        localStorage.setItem('token', resp.data.token);
```

Comment:
Change this from local storage to using authService as soon as you can. We won't use local storage in webapp